superset/app.py

```diff
@@ -36,6 +37,10 @@ def create_app() -> Flask:
         app_initializer = app.config.get("APP_INITIALIZER", SupersetAppInitializer)(app)
         app_initializer.init_app()
 
+        profiling = app.config.get("PROFILING", False)
+        if profiling:
```

Comment:
```suggestion
        if app.config[“PROFILING”]:
```

No need to use `dict.get(…)` since the key is guaranteed to exist.